.github/actions/check_docstrings/action.yml

```diff
@@ -0,0 +1,63 @@
+name: Check Docstrings
+inputs:
+    python-version:
+        description: 'The version of Python to use for the workflow.'
+        default: '3.10'
+        required: false
+        type: string
+    repository:
+        description: 'The repository to check the docstrings for.'
+        required: True
+        type: string
+    branch:
+        description: 'The branch to check the docstrings for.'
+        required: false
+        type: string
+        default: ''
+    package-name:
+        description: 'The name of the package to check the docstrings for.'
+        required: True
+        type: string
+    names-to-skip:
+        description: 'The names of the modules/classes/functions/methods to skip in the docstring check.'
+        required: false
+        type: string
+        default: ''
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Conda
+      uses: s-weigand/setup-conda@v1
```

Comment:
Just curious - do we actually need this step?

Reply:
+1 on this question. I don't see where this is needed.